src/codec/liquiditypool/liquiditypool.ts

```diff
@@ -414,12 +386,7 @@ export const Pool = {
     message.amountB = object.amountB ?? "";
     message.weightB = object.weightB ?? "";
     message.swapFee = object.swapFee ?? "";
-    message.numQuotes =
-      object.numQuotes !== undefined && object.numQuotes !== null
-        ? Long.fromValue(object.numQuotes)
-        : Long.ZERO;
     message.sharesAmount = object.sharesAmount ?? "";
-    message.market = object.market ?? "";
     message.ampBps =
       object.ampBps !== undefined && object.ampBps !== null
         ? Long.fromValue(object.ampBps)
```

Comment:
May need to remove numQuotes and market fields on Pool struct on `WSModels`.
<img width="832" alt="Screenshot 2023-08-07 at 12 09 32 PM" src="https://github.com/Switcheo/carbon-js-sdk/assets/65583522/dcf2f399-3f58-4267-abb8-210102c468fd">